pyproject.toml

```diff
@@ -146,3 +121,30 @@ known_first_party = ["commitizen", "tests"]
 [build-system]
 requires = ["poetry_core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.pytest.ini_options]
+addopts = "--strict-markers"
+
+[tool.ruff]
+line-length = 88
+ignore = [
+    "E501",
+    "D1",
+    "D415",
+]
+
+[tool.ruff.isort]
+known-first-party = ["commitizen", "tests"]
+
+[tool.ruff.pydocstyle]
+convention = "google"
+
+[tool.mypy]
+files = "commitizen"
+ignore_missing_imports = true
+disallow_untyped_decorators = true
+disallow_subclassing_any = true
+warn_return_any = true
+warn_redundant_casts = true
+warn_unused_ignores = true
+warn_unused_configs = true
```

Comment:
I'm not sure but won't it be too strict for our codebase now? but I'd be really glad if commitizen can pass these checks 🙌

Reply:
I think this is how it was configured (and they are passing). But mypy settings where in the `setup.cfg` before.